test/elixir_bench/benchmarks/benchmarks_test.exs

```diff
@@ -0,0 +1,18 @@
+defmodule ElixirBench.BenchmarksTest do
+  use ElixirBench.DataCase
+  alias ElixirBench.{Benchmarks, Benchmarks.Job}
+  alias ElixirBench.Repos
+
+  describe "when correct params" do
+    test "create_job/2 should return new Job" do
+      {:ok, repo} = Repos.create_repo(%{owner: "elixir-ecto", name: "ecto"})
+      job_attrs = %{branch_name: "mm/benche", commit_sha: "ABC123"}
+
+      jobs_count = ElixirBench.Repo.all(Job) |> length
```

Comment:
you can do `Repo.aggregate(Job, :count, :id)` iirc to just get the count directly from the db :)